include/bout/solver.hxx

```diff
@@ -459,6 +459,9 @@ private:
   /// Diffusive part (if split operator)
   rhsfunc phys_diff{nullptr};
 
+  /// Should unsplit physics models be treated as diffusive?
+  bool unsplit_diffusive{true};
```

Comment:
How do you feel about `is_unsplit_diffusive`, or even `is_nonsplit_model_diffusive`?

Reply:
yes happy with either of those. Maybe `is_nonsplit_model_diffusive` is clearer?